libavcodec/jpeg2000htdec.c

```diff
@@ -406,14 +406,15 @@ static void recover_mag_sgn(StateVars *mag_sgn, uint8_t pos, uint16_t q, int32_t
             E[n] = 32 - ff_clz(v[pos][i] | 1);
             mu_n[n] = (v[pos][i] >> 1) + 1;
             mu_n[n] <<= pLSB;
+            mu_n[n] |= (1 << (pLSB - 1)); // Add 0.5 (reconstruction parameter = 1/2)
             mu_n[n] |= ((uint32_t) (v[pos][i] & 1)) << 31; // sign bit.
         }
     }
 }
 
 static int jpeg2000_import_bit(StateVars *stream, const uint8_t *array, uint32_t length)
 {
-    int cond = stream->pos < length;
+    int cond = stream->pos <= length;
```

Comment:
Didn't we just change it in https://github.com/FFmpeg/FFmpeg/commit/249c66bb225b0671434b3ce9cc3f7935a229f428 ?

Reply:
This is just a leftover. In the next commit, this change should disappear.